src/build.py

```diff
@@ -825,6 +825,15 @@ def OverrideCMakeCompiler():
           '-DCMAKE_CXX_COMPILER=' + CXX]
 
 
+def CMakeCommand(args):
+  command = [PREBUILT_CMAKE_BIN, '-G', 'Ninja'] + args
+  # Python's location could change, so always update CMake's cache
+  command.append('-DPYTHON_EXECUTABLE=%s' % proc.Which('python', os.getcwd()))
+  command.extend(OverrideCMakeCompiler())
```

Comment:
So there are really 2 different types of builds we do: native builds, and cross-builds using the wasm compiler. It looks like Python is really the only thing that they have in common (edit: also I guess `CMAKE_BUILD_TYPE`). `OverrideCMakeCompiler` only applies to native builds (for wasm builds `CC` is set by the toolchain file). Also Goma only works for native builds (that's what the `CMakeLauncherFlags` is for). 
So maybe it makes sense to just have 2 different `CMakeCommand` functions, i.e. `CMakeCommandNative` and `CMakeCommandWasmClang` or (`CMakeCommandWack`). Then `OverrideCMakeCompiler`, `CMakeLauncherFlags`, and `CMAKE_INSTALL_PREFIX` can go in the native one, and `CMAKE_TOOLCHAIN_FILE` can go in the wasm one.
Come to think of it, I'm not even sure Python is used by any of the wasm builds, but I guess it wouldn't hurt to keep it in both just to be safe.